refinery/core/serializers.py

```diff
@@ -121,26 +118,6 @@ def _get_file_extension(self, obj):
     def _get_relative_url(self, obj):
         return obj.get_relative_file_store_item_url() or None
 
-    def _ready_for_igv_detail_view(self, obj):
```

Comment:
I think this method may still be needed, but this is just off the top of my head.